packages/documentation-v7/src/stories/foundation/layout/breakpoints/breakpoints.docs.mdx

```diff
@@ -0,0 +1,86 @@
+import { Meta, Story } from '@storybook/blocks';
+import { BADGE } from '../../../../../.storybook/constants';
+import { BreakpointTable, SCSS_VARIABLES } from './breakpoints.blocks';
+
+<Meta title="Foundations/Layout/Breakpoints" parameters={{ badges: [BADGE.NEEDS_REVISION] }} />
+
+# Breakpoints
+
+<div class="lead">
+  Breakpoints are widths that determine how the responsive layout behaves across device or viewport
+  sizes.
+</div>
+
+## Available breakpoints
+
+The SwissPost Design-System includes {Object.keys(SCSS_VARIABLES.breakpoint).length} breakpoints, sometimes referred to as grid tiers, for building responsively. These breakpoints can be customized if you’re using our source Sass files.
+
+<BreakpointTable />
+
+You’ll find these breakpoints in a Sass map in our styles core.
+
+```scss
+@use '@swisspost/design-system-styles/core' as post;
+
+@each $key, $value in post.$grid-breakpoints {
+  ...
+}
+```
+
+## Mixins
+
+There are several mixins available to use in your own Sass to help with building responsive layouts.
+
+### Min-width
+
+We primarily use the following media query ranges—or breakpoints—in our source Sass files for our layout, grid system, and components.
+
+```scss
+// No media query necessary for xs breakpoint as it's effectively @media (min-width: 0) { ... }
+.custom-class {
+  display: none;
+}
+
+@include media-breakpoint-up(lg) {
+  .custom-class {
+    display: block;
+  }
+}
+```
+
+### Max-width
+
+We occasionally use media queries that go in the other direction.
+
+```scss
+// No media query necessary for xs breakpoint as it's effectively @media (max-width: 0) { ... }
+@include media-breakpoint-down(lg) {
```

Comment:
In projects this mixins should be used with the `post` namespace. Otherwise you get a "SassError: Undefined mixin."

```suggestion
@include post.media-breakpoint-down(lg) {
```